compose.yaml

```diff
@@ -37,24 +37,6 @@ services:
     stdin_open: true
     tty: true
 
-  osh-worker:
```

Comment:
I have temporarily disabled running worker in a container for testing purpose. This would not be in the final pull request.